csrc/Makefile.osx

```diff
@@ -9,7 +9,8 @@ libmapnik-jni.jnilib: $(SOURCE_DEPENDS)
 		$(JAVA_CFLAGS) \
 		mapnikjni.cpp \
 		$(LDFLAGS) \
-		$(MAPNIK_LIBS) -framework JavaVM
```

Comment:
The `JavaVM.framework` contains symlinks to a non-existent JDK 1.6 on my Mac; I'm not sure if that is the default state of Macs nowadays, or if I broke things by installing and uninstalling JDKs.